src/Arc.StringSanitizer/Arc.StringSanitizer.csproj

```diff
@@ -34,4 +33,8 @@
     <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0' Or '$(TargetFramework)' == 'netstandard2.1'">
+    <LangVersion>13</LangVersion>
```

Comment:
Previously the lang version was `11`, I have updated it to `13` which is the latest version.